### PR TITLE
Set SortingHat as a required package in grimoireelk

### PR DESCRIPTION
**Pull request by jjmerchante**
SortingHat was set as an extra dependency in elk, but actually it is required for running elk.

This PR updates the Poetry dependencies to make it a required package and updates the GitHub actions to remove Sortinghat as an extra dependency.

### CODE REVIEW

**Comment by sduenas** [comment]
Related to: https://github.com/chaoss/grimoirelab/issues/489

**Review by sduenas — CHANGES_REQUESTED** [review]
LGTM but you should update the dependencies. They were updated in the last PR we merged. Thanks!

**Comment by jjmerchante** [comment]
PR rebased and dependencies updated

File: releases/unreleased/set-sortinghat-as-a-required-package-in-elk.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Set SortingHat as a required package in elk
```
Comment (sduenas) [inline releases/unreleased/set-sortinghat-as-a-required-package-in-elk.yml:2]: I think this title is better for the changelog.

```suggestion
title: SortingHat as a required package
```

**Review by sduenas — CHANGES_REQUESTED** [review]
LGTM but check my comment if it makes sense to you.

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**